deps/wazuh_testing/wazuh_testing/event_monitor.py

```diff
@@ -18,22 +18,22 @@ def make_callback(pattern, prefix=''):
 
 
 def check_event(file_monitor=None, callback='', error_message=None, update_position=True, timeout=20,
-                accum_results=1, file_to_monitor=None):
+                accum_results=1, file_to_monitor=None, prefix=None):
```

Comment:
Add `file_to_monitor` to function documentation

Reply:
Done in 56061f4